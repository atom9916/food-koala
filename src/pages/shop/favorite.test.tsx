import '@testing-library/jest-dom/extend-expect';
import ShopFavorite from './favorite';
import { render } from '@testing-library/react';

// describe('Snapshot', () => {
//   // ログイン時のスナップショットテスト
//   it('Snapshot test of favorite_page_login', () => {
//     const favorite = `    <>
//         <Head>
//           <title>お気に入り店舗一覧</title>
//         </Head>
//         <Header />
//         <BreadList list={[menu_list, favorite_list]} />
//         <ShopName data={favoriteShops} />
//         <Footer />
//       </>`;
//     expect(favorite).toMatchSnapshot();
//   });
//   // ログイン前のスナップショットテスト
//   it('Snapshot test of favorite_page_logout', () => {
//     const favorite_logout = `<>
//     <Header />
//     <BreadList list={[menu_list, favorite_list]} />
//     <div>
//       <div className={styles.favorite_login}>
//         <div className={styles.favorite_login_link}>
//           <img src="/images/foodkoala_img2.png" alt="コアラ" />
//           <br />
//           <br />
//           <a href="/login">ログイン</a>
//         </div>
//         <br />
//         <p>
//           お気に入り店舗一覧を表示したい場合はログインをしてください
//         </p>
//       </div>
//     </div>
//     <Footer />
//   </>`;
//     expect(favorite_logout).toMatchSnapshot();
//   });
//   // ログイン後お気に入り登録がない場合のスナップショットテスト
//   it('Snapshot test of no_favorite_page', () => {
//     const no_favorite_page = `<>
//     <Header />
//     <BreadList list={[menu_list, favorite_list]} />
//     <div>
//       <div className={styles.favorite_login}>
//         <div className={styles.favorite_login_link}>
//           <img src="/images/foodkoala_img2.png" alt="コアラ" />
//           <br />
//           <br />
//           <a href="/shop/list">ショップ一覧へ</a>
//         </div>
//         <br />
//         <p>お気に入り店舗がありません</p>
//       </div>
//     </div>
//     <Footer />
//   </>`;
//     expect(no_favorite_page).toMatchSnapshot();
//   });
// });

// describe('favoriteスナップショット', () => {
//   const { asFragment } = render(<ShopFavorite />);
//   expect(asFragment()).toMatchSnapshot();
// });
