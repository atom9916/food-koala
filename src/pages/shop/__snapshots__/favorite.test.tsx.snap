// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Snapshot test of favorite_page_login 1`] = `
"    <>
        <Head>
          <title>お気に入り店舗一覧</title>
        </Head>
        <Header />
        <BreadList list={[menu_list, favorite_list]} />
        <ShopName data={favoriteShops} />
        <Footer />
      </>"
`;

exports[`Snapshot Snapshot test of favorite_page_logout 1`] = `
"<>
    <Header />
    <BreadList list={[menu_list, favorite_list]} />
    <div>
      <div className={styles.favorite_login}>
        <div className={styles.favorite_login_link}>
          <img src="/images/foodkoala_img2.png" alt="コアラ" />
          <br />
          <br />
          <a href="/login">ログイン</a>
        </div>
        <br />
        <p>
          お気に入り店舗一覧を表示したい場合はログインをしてください
        </p>
      </div>
    </div>
    <Footer />
  </>"
`;

exports[`Snapshot Snapshot test of no_favorite_page 1`] = `
"<>
    <Header />
    <BreadList list={[menu_list, favorite_list]} />
    <div>
      <div className={styles.favorite_login}>
        <div className={styles.favorite_login_link}>
          <img src="/images/foodkoala_img2.png" alt="コアラ" />
          <br />
          <br />
          <a href="/shop/list">ショップ一覧へ</a>
        </div>
        <br />
        <p>お気に入り店舗がありません</p>
      </div>
    </div>
    <Footer />
  </>"
`;

exports[`お気に入りページのテスト データ待ちの場合のスナップショット 1`] = `[Function]`;

exports[`お気に入りページのテスト ログインしていない場合のスナップショット 1`] = `[Function]`;

exports[`お気に入りページのテスト ログイン後お気に入りがある場合のスナップショットテスト 1`] = `[Function]`;

exports[`お気に入りページのテスト ログイン後お気に入りがない場合のスナップショットテスト 1`] = `[Function]`;
